ios/FluentUI/Vnext/Persona/PersonaView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public protocol PersonaCellProtocol {
```

Comment:
The name of the protocol can be PersonaViewState. And the state implementation can be called PersonaViewStateImpl